android/src/androidTest/java/com/reactnativestripesdk/paymentsheet/PaymentSheetFragmentTest.kt

```diff
@@ -8,6 +8,13 @@ import org.junit.Assert
 import org.junit.Test
 
 class PaymentSheetFragmentTest {
+  @Test
+  fun hasNoArgsConstructor() {
+    // This is just a type check to make sure to keep a no args constructor
+    // so that when the fragment is re-created it doesn't crash.
+    PaymentSheetFragment()
```

Comment:
Should we add tests like this for the other fragments too?

Reply:
Would be good, I only set it up in that one since it already had a test file, but I could create it for all other ones too.

Reply:
Added tests for all fragments